src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -1992,6 +1992,8 @@ public void resumeBootstrap(PrintStream out) throws IOException
             {
                 out.println("Resuming bootstrap");
                 monitor.awaitCompletion();
+                if (monitor.getError() != null)
+                    throw monitor.getError();
```

Comment:
This is obviously going to change the return contract for `resume`, but it's really just a bug fix. I don't see it requiring a `NEWS.txt` entry. (Just `CHANGES.txt`.)